pom.xml

```diff
@@ -120,6 +120,32 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>release-sign-artifacts</id>
```

Comment:
Would it instead make sense to change the name of the existing `release` profile to `release-sign-artifacts`, and add your updates into it?
https://github.com/roflinn/jenatools/blob/08c5f0078e28ba667af6d6632a25ad6da9821a93/pom.xml#L79